docs/self-managed/operational-guides/update-guide/840-to-850.md

```diff
@@ -19,3 +19,15 @@ The broker health check routes have moved, and the old routes are now deprecated
 | http://{zeebe-broker-host}:9600/startup | **http://{zeebe-broker-host}:9600/actuator/health/startup**   |
 
 Please migrate to the new routes in your deployments. **If you're using the official Helm charts, then you don't have to do anything here.**
+
+### Changes to exported records
+
+The `UserTask` events like `UserTask:CREATED` don't export the string value attributes `candidateUsers` and `candidateGroups` anymore.
+As a replacement, user task events now feature the string array attributes `candidateUsersList` and `candidateGroupsList`.
+Custom exporters using these events must be modified accordingly.
+
+:::warning Breaking Change
+The values of the `candidateUsers` and `candidateGroups` attributes in user task records are lost during an update from `8.4` to `8.5`.
+There is no data migration that moves the values from `candidateUsers` and `candidateGroups` to `candidateUsersList` and `candidateGroupsList`
+since the Zeebe user tasks are only experimental in `8.4`.
+:::
```

Comment:
🔧 You could mention that users that need to have these properties on the user tasks, can re-activate the user task and evaluate the expressions again using [Process Instance Modification](https://docs.camunda.io/docs/next/components/concepts/process-instance-modification/). This should result in `UserTask:CANCELED` events for the existing user task and new `UserTask:CREATED` events with the `candidateUsersList` and `candidateGroupsList` filled as expected.

Reply:
Uh, that's a neat little trick, I like it! 💡